gef.py

```diff
@@ -7909,6 +7909,7 @@ def do_invoke(self, argv):
         reverse = False
 
         for arg in argv:
+            arg_old = arg   # if it were to be assigned to target, we don't want to convert it to lower case
             arg = arg.lower()
```

Comment:
It's probably cleaner to use a different variable name during the `.lower()` affectation (i.e instead of `arg = arg.lower()`) and  process with that one. Can you change that?